StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -112,25 +155,44 @@ class RowButton: UIView {
     }
 #endif
 
-    func updateAccessibilityTraits() {
-        var traits: UIAccessibilityTraits = [.button]
-        if isSelected {
-            traits.insert(.selected)
+    // MARK: Overridable functions
+
+    /// Override this function to setup the UI for your RowButton subclass
+    func setupUI() {
+        stpAssertionFailure("RowButton init not called from subclass, use RowButton.create() instead of RowButton(...).")
```

Comment:
I don't love that anyone can call RowButton's init, they should only call it on the subclasses. If they do they'll hit this assert, but I can't find a way to make a compile time check. Making the init private renders it not available to the sub-classes, so this might be a tradeoff.